src/_nebari/upgrade.py

```diff
@@ -1179,8 +1237,9 @@ def _version_specific_upgrade(
                     },
                     indent=4,
                 )
-                text += "\n\nHit enter to continue"
-                Prompt.ask(text)
+                rich.print(text)
+                if not attempt_fixes:
```

Comment:
Same as above: we don't need `attempt_fixes` here as it will only run if it's `False` and at that point we don't really care